fargate/upload-move/pkg/AWSRegions.go

```diff
@@ -0,0 +1,44 @@
+package pkg
+
+type AWSRegions struct {
+	FullName   string
+	RegionCode string
+}
+
+// Regions as of Feb 2025
+var Regions = map[string]AWSRegions{
```

Comment:
with the other one being singular you can call this one `AWSRegions` for more clarity